
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://lambda-ber.github.io/lambda-ber-schema/">
      
      
      
        <link rel="next" href="elements/">
      
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.22">
    
    
      
        <title>lambda-ber-schema</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.84d31ad4.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="green">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#lambda-ber-schema-schema" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="lambda-ber-schema" class="md-header__button md-logo" aria-label="lambda-ber-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            lambda-ber-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Home
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="green"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="amber" data-md-color-accent="green"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/lambda-ber/lambda-ber-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="lambda-ber-schema" class="md-nav__button md-logo" aria-label="lambda-ber-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    lambda-ber-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/lambda-ber/lambda-ber-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#schema-organization" class="md-nav__link">
    <span class="md-ellipsis">
      Schema Organization
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Schema Organization">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#biological-materials" class="md-nav__link">
    <span class="md-ellipsis">
      Biological Materials
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#data-collection" class="md-nav__link">
    <span class="md-ellipsis">
      Data Collection
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#data-processing" class="md-nav__link">
    <span class="md-ellipsis">
      Data Processing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#data-products" class="md-nav__link">
    <span class="md-ellipsis">
      Data Products
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-usage" class="md-nav__link">
    <span class="md-ellipsis">
      Example Usage
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#key-features" class="md-nav__link">
    <span class="md-ellipsis">
      Key Features
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="elements/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Schema
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Background
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Background
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/nexus/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    NEXUS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/mmcif/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    mmCIF
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/empiar/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    EMPIAR
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/emdb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    EMDB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/dials/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    DIALS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/sasdb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SASDB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/rembi-alignment/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    REMBI
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/lambda-ber-schema-data-lakehouse-strategy/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Lakehouse Strategy
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/ber-doe-facilities/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Integration with BER
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="background/data-quality-challenges/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Quality Challenges
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Examples
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Examples
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="examples/simplescattering-glurs-analysis/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Simple Scattering GluRS Dataset
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Technical
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Technical
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="technical/ome-mapping/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mapping to OME
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Slides
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Slides
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="slides/overview.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="slides/technical-overview.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Technical Deep Dive
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#schema-organization" class="md-nav__link">
    <span class="md-ellipsis">
      Schema Organization
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Schema Organization">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#biological-materials" class="md-nav__link">
    <span class="md-ellipsis">
      Biological Materials
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#data-collection" class="md-nav__link">
    <span class="md-ellipsis">
      Data Collection
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#data-processing" class="md-nav__link">
    <span class="md-ellipsis">
      Data Processing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#data-products" class="md-nav__link">
    <span class="md-ellipsis">
      Data Products
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-usage" class="md-nav__link">
    <span class="md-ellipsis">
      Example Usage
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#key-features" class="md-nav__link">
    <span class="md-ellipsis">
      Key Features
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      Classes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      Slots
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      Types
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="lambda-ber-schema-schema">lambda-ber-schema-schema</h1>
<p>lambda-ber-schema is a comprehensive schema for representing multimodal structural biology imaging data, 
from atomic-resolution structures to tissue-level organization. It supports diverse experimental 
techniques including cryo-EM, X-ray crystallography, SAXS/SANS, fluorescence microscopy, and 
spectroscopic imaging.</p>
<p><strong>NOTE</strong> this schema was developed rapidly using AI assistance, there may be mistakes!</p>
<h2 id="schema-organization">Schema Organization</h2>
<p>The schema follows a hierarchical structure that mirrors how structural biology research is organized:</p>
<p>The top-level entity is a <a href="elements/Dataset/">Dataset</a>, which serves as a container for related research.
A dataset might represent all data from a specific grant, collaboration, or publication.</p>
<p>Each dataset contains one or more <a href="elements/Study/">Studies</a>, which are focused investigations of specific
biological questions. For example, a study might investigate "Heat stress response in Arabidopsis"
or "Structure of the human ribosome under different conditions."</p>
<p>Within each study, you'll find:</p>
<h3 id="biological-materials">Biological Materials</h3>
<ul>
<li>
<p><a href="elements/Sample/">Samples</a>: The biological specimens being studied (proteins, nucleic acids, complexes, 
  cells, tissues). Each sample includes detailed molecular composition, buffer conditions, and 
  storage information. For example, a purified protein with its sequence, concentration, and buffer pH.</p>
</li>
<li>
<p><a href="elements/SamplePreparation/">Sample Preparations</a>: How samples were prepared for specific techniques.
  This includes cryo-EM grid preparation (vitrification parameters), crystallization conditions for
  X-ray studies, or staining protocols for fluorescence microscopy.</p>
</li>
</ul>
<h3 id="data-collection">Data Collection</h3>
<ul>
<li>
<p><a href="elements/Instrument/">Instruments</a>: The equipment used, from Titan Krios microscopes to synchrotron 
  beamlines. Each instrument type (<a href="elements/CryoEMInstrument/">CryoEMInstrument</a>, 
  <a href="elements/XRayInstrument/">XRayInstrument</a>, <a href="elements/SAXSInstrument/">SAXSInstrument</a>) has specific parameters
  like accelerating voltage, detector type, or beam energy.</p>
</li>
<li>
<p><a href="elements/ExperimentRun/">Experiment Runs</a>: Individual data collection sessions that link samples to 
  instruments. An experiment run captures when, how, and under what conditions data was collected,
  including quality metrics like resolution and completeness.</p>
</li>
</ul>
<h3 id="data-processing">Data Processing</h3>
<ul>
<li><a href="elements/WorkflowRun/">Workflow Runs</a>: Computational processing steps applied to raw data. This includes
  motion correction for cryo-EM movies, 3D reconstruction, model building, or phase determination
  for crystallography. Each workflow tracks the software used, parameters, and computational resources.</li>
</ul>
<h3 id="data-products">Data Products</h3>
<ul>
<li>
<p><a href="elements/DataFile/">Data Files</a>: Any files generated or used, from raw data to final models. Each file
  is tracked with checksums for data integrity and typed (micrograph, particles, volume, model).</p>
</li>
<li>
<p><a href="elements/Image/">Images</a>: Specialized classes for different imaging modalities:</p>
</li>
<li><a href="elements/Image2D/">Image2D</a>: Micrographs, diffraction patterns</li>
<li><a href="elements/Image3D/">Image3D</a>: 3D reconstructions, tomograms</li>
<li><a href="elements/FTIRImage/">FTIRImage</a>: Molecular composition maps from infrared spectroscopy</li>
<li><a href="elements/FluorescenceImage/">FluorescenceImage</a>: Fluorophore-labeled cellular components</li>
<li><a href="elements/OpticalImage/">OpticalImage</a>: Brightfield/phase contrast microscopy</li>
<li><a href="elements/XRFImage/">XRFImage</a>: Elemental distribution maps</li>
</ul>
<h2 id="example-usage">Example Usage</h2>
<p>A typical cryo-EM study of a protein complex would include:</p>
<ol>
<li>Sample records for the purified complex with molecular weight and buffer composition</li>
<li>Grid preparation details with vitrification parameters</li>
<li>Microscope specifications and data collection parameters</li>
<li>Processing workflows from motion correction through 3D refinement</li>
<li>Final reconstructed volumes and fitted atomic models</li>
</ol>
<p>A multimodal plant imaging study might combine:</p>
<ol>
<li>Whole plant optical imaging for morphology</li>
<li>XRF imaging to map nutrient distribution</li>
<li>FTIR spectroscopy to identify stress-related molecular changes</li>
<li>Fluorescence microscopy to track specific protein responses</li>
<li>Cryo-EM of isolated organelles for ultrastructural details</li>
</ol>
<h2 id="key-features">Key Features</h2>
<ul>
<li><strong>Technique-agnostic core</strong>: The same schema handles data from any structural biology method</li>
<li><strong>Rich metadata</strong>: Comprehensive tracking from sample to structure</li>
<li><strong>Workflow provenance</strong>: Complete computational reproducibility</li>
<li><strong>Multimodal support</strong>: Seamlessly integrate data across scales and techniques</li>
<li><strong>Standards-compliant</strong>: Follows FAIR principles and integrates with existing ontologies</li>
</ul>
<p>URI: https://w3id.org/lambda-ber-schema/</p>
<p>Name: lambda-ber-schema-schema</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="elements/BufferComposition/">BufferComposition</a></td>
<td>Buffer composition for sample storage</td>
</tr>
<tr>
<td><a href="elements/ComputeResources/">ComputeResources</a></td>
<td>Computational resources used</td>
</tr>
<tr>
<td><a href="elements/DataCollectionStrategy/">DataCollectionStrategy</a></td>
<td>Strategy for data collection</td>
</tr>
<tr>
<td><a href="elements/ExperimentalConditions/">ExperimentalConditions</a></td>
<td>Environmental and experimental conditions</td>
</tr>
<tr>
<td><a href="elements/ImageFeature/">ImageFeature</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/MolecularComposition/">MolecularComposition</a></td>
<td>Molecular composition of a sample</td>
</tr>
<tr>
<td><a href="elements/NamedThing/">NamedThing</a></td>
<td>A named thing</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/DataFile/">DataFile</a></td>
<td>A data file generated or used in the study</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Dataset/">Dataset</a></td>
<td>A collection of studies</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/ExperimentRun/">ExperimentRun</a></td>
<td>An experimental data collection session</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Image/">Image</a></td>
<td>An image file from structural biology experiments</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/FTIRImage/">FTIRImage</a></td>
<td>Fourier Transform Infrared (FTIR) spectroscopy image capturing molecular comp...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Image2D/">Image2D</a></td>
<td>A 2D image (micrograph, diffraction pattern)</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/FluorescenceImage/">FluorescenceImage</a></td>
<td>Fluorescence microscopy image capturing specific molecular targets through fl...</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/OpticalImage/">OpticalImage</a></td>
<td>Visible light optical microscopy or photography image</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/XRFImage/">XRFImage</a></td>
<td>X-ray fluorescence (XRF) image showing elemental distribution</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Image3D/">Image3D</a></td>
<td>A 3D volume or tomogram</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Instrument/">Instrument</a></td>
<td>An instrument used to collect data</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/CryoEMInstrument/">CryoEMInstrument</a></td>
<td>Cryo-EM microscope specifications</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/SAXSInstrument/">SAXSInstrument</a></td>
<td>SAXS/WAXS instrument specifications</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/XRayInstrument/">XRayInstrument</a></td>
<td>X-ray diffractometer or synchrotron beamline specifications</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Sample/">Sample</a></td>
<td>A biological sample used in structural biology experiments</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/SamplePreparation/">SamplePreparation</a></td>
<td>A process that prepares a sample for imaging</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/Study/">Study</a></td>
<td></td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/WorkflowRun/">WorkflowRun</a></td>
<td>A computational processing workflow execution</td>
</tr>
<tr>
<td><a href="elements/OntologyTerm/">OntologyTerm</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/QualityMetrics/">QualityMetrics</a></td>
<td>Quality metrics for experiments</td>
</tr>
<tr>
<td><a href="elements/StorageConditions/">StorageConditions</a></td>
<td>Storage conditions for samples</td>
</tr>
<tr>
<td><a href="elements/TechniqueSpecificPreparation/">TechniqueSpecificPreparation</a></td>
<td>Base class for technique-specific preparation details</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/CryoEMPreparation/">CryoEMPreparation</a></td>
<td>Cryo-EM specific sample preparation</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/SAXSPreparation/">SAXSPreparation</a></td>
<td>SAXS/WAXS specific preparation</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="elements/XRayPreparation/">XRayPreparation</a></td>
<td>X-ray crystallography specific preparation</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="elements/accelerating_voltage/">accelerating_voltage</a></td>
<td>Accelerating voltage in kV</td>
</tr>
<tr>
<td><a href="elements/acquisition_date/">acquisition_date</a></td>
<td>Date image was acquired</td>
</tr>
<tr>
<td><a href="elements/additives/">additives</a></td>
<td>Additional additives in the buffer</td>
</tr>
<tr>
<td><a href="elements/apodization_function/">apodization_function</a></td>
<td>Mathematical function used for apodization</td>
</tr>
<tr>
<td><a href="elements/astigmatism/">astigmatism</a></td>
<td>Astigmatism value</td>
</tr>
<tr>
<td><a href="elements/atmosphere/">atmosphere</a></td>
<td>Storage atmosphere conditions</td>
</tr>
<tr>
<td><a href="elements/autoloader_capacity/">autoloader_capacity</a></td>
<td>Number of grids the autoloader can hold</td>
</tr>
<tr>
<td><a href="elements/background_correction/">background_correction</a></td>
<td>Method used for background correction</td>
</tr>
<tr>
<td><a href="elements/beam_energy/">beam_energy</a></td>
<td>X-ray beam energy in keV</td>
</tr>
<tr>
<td><a href="elements/beam_size/">beam_size</a></td>
<td>X-ray beam size in micrometers</td>
</tr>
<tr>
<td><a href="elements/beam_size_max/">beam_size_max</a></td>
<td>Maximum beam size in micrometers</td>
</tr>
<tr>
<td><a href="elements/beam_size_min/">beam_size_min</a></td>
<td>Minimum beam size in micrometers</td>
</tr>
<tr>
<td><a href="elements/blot_force/">blot_force</a></td>
<td>Blotting force setting</td>
</tr>
<tr>
<td><a href="elements/blot_time/">blot_time</a></td>
<td>Blotting time in seconds</td>
</tr>
<tr>
<td><a href="elements/buffer_composition/">buffer_composition</a></td>
<td>Buffer composition including pH, salts, additives</td>
</tr>
<tr>
<td><a href="elements/buffer_matching_protocol/">buffer_matching_protocol</a></td>
<td>Protocol for buffer matching</td>
</tr>
<tr>
<td><a href="elements/calibration_standard/">calibration_standard</a></td>
<td>Reference standard used for calibration</td>
</tr>
<tr>
<td><a href="elements/cell_path_length/">cell_path_length</a></td>
<td>Path length in mm</td>
</tr>
<tr>
<td><a href="elements/chamber_temperature/">chamber_temperature</a></td>
<td>Chamber temperature in Celsius</td>
</tr>
<tr>
<td><a href="elements/channel_name/">channel_name</a></td>
<td>Name of the fluorescence channel (e</td>
</tr>
<tr>
<td><a href="elements/checksum/">checksum</a></td>
<td>SHA-256 checksum for data integrity</td>
</tr>
<tr>
<td><a href="elements/collection_mode/">collection_mode</a></td>
<td>Mode of data collection</td>
</tr>
<tr>
<td><a href="elements/color_channels/">color_channels</a></td>
<td>Color channels present (e</td>
</tr>
<tr>
<td><a href="elements/completed_at/">completed_at</a></td>
<td>Workflow completion time</td>
</tr>
<tr>
<td><a href="elements/completeness/">completeness</a></td>
<td>Data completeness percentage</td>
</tr>
<tr>
<td><a href="elements/components/">components</a></td>
<td>Buffer components and their concentrations</td>
</tr>
<tr>
<td><a href="elements/compute_resources/">compute_resources</a></td>
<td>Computational resources used</td>
</tr>
<tr>
<td><a href="elements/concentration/">concentration</a></td>
<td>Sample concentration in mg/mL or µM</td>
</tr>
<tr>
<td><a href="elements/concentration_series/">concentration_series</a></td>
<td>Concentration values for series measurements</td>
</tr>
<tr>
<td><a href="elements/concentration_unit/">concentration_unit</a></td>
<td>Unit of concentration measurement</td>
</tr>
<tr>
<td><a href="elements/contrast_method/">contrast_method</a></td>
<td>Contrast enhancement method used</td>
</tr>
<tr>
<td><a href="elements/cpu_hours/">cpu_hours</a></td>
<td>CPU hours used</td>
</tr>
<tr>
<td><a href="elements/creation_date/">creation_date</a></td>
<td>File creation date</td>
</tr>
<tr>
<td><a href="elements/cryoprotectant/">cryoprotectant</a></td>
<td>Cryoprotectant used</td>
</tr>
<tr>
<td><a href="elements/cryoprotectant_concentration/">cryoprotectant_concentration</a></td>
<td>Cryoprotectant concentration percentage</td>
</tr>
<tr>
<td><a href="elements/crystal_cooling_capability/">crystal_cooling_capability</a></td>
<td>Crystal cooling system available</td>
</tr>
<tr>
<td><a href="crystal_size.md">crystal_size</a></td>
<td>Crystal dimensions in micrometers</td>
</tr>
<tr>
<td><a href="elements/crystallization_conditions/">crystallization_conditions</a></td>
<td>Detailed crystallization conditions</td>
</tr>
<tr>
<td><a href="elements/crystallization_method/">crystallization_method</a></td>
<td>Method used for crystallization</td>
</tr>
<tr>
<td><a href="elements/cs_corrector/">cs_corrector</a></td>
<td>Spherical aberration corrector present</td>
</tr>
<tr>
<td><a href="elements/current_status/">current_status</a></td>
<td>Current operational status</td>
</tr>
<tr>
<td><a href="elements/data_collection_strategy/">data_collection_strategy</a></td>
<td>Strategy for data collection</td>
</tr>
<tr>
<td><a href="elements/data_files/">data_files</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/data_type/">data_type</a></td>
<td>Type of data in the file</td>
</tr>
<tr>
<td><a href="elements/definition/">definition</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/defocus/">defocus</a></td>
<td>Defocus value in micrometers</td>
</tr>
<tr>
<td><a href="elements/description/">description</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/detector_dimensions/">detector_dimensions</a></td>
<td>Detector dimensions in pixels (e</td>
</tr>
<tr>
<td><a href="elements/detector_distance_max/">detector_distance_max</a></td>
<td>Maximum detector distance in mm</td>
</tr>
<tr>
<td><a href="elements/detector_distance_min/">detector_distance_min</a></td>
<td>Minimum detector distance in mm</td>
</tr>
<tr>
<td><a href="detector_type.md">detector_type</a></td>
<td>Type of detector</td>
</tr>
<tr>
<td><a href="elements/dimensions_x/">dimensions_x</a></td>
<td>Image width in pixels</td>
</tr>
<tr>
<td><a href="elements/dimensions_y/">dimensions_y</a></td>
<td>Image height in pixels</td>
</tr>
<tr>
<td><a href="elements/dimensions_z/">dimensions_z</a></td>
<td>Image depth in pixels/slices</td>
</tr>
<tr>
<td><a href="elements/dose/">dose</a></td>
<td>Electron dose in e-/Å²</td>
</tr>
<tr>
<td><a href="elements/dose_per_frame/">dose_per_frame</a></td>
<td>Dose per frame</td>
</tr>
<tr>
<td><a href="elements/duration/">duration</a></td>
<td>Storage duration</td>
</tr>
<tr>
<td><a href="elements/dwell_time/">dwell_time</a></td>
<td>Dwell time per pixel in milliseconds</td>
</tr>
<tr>
<td><a href="elements/elements_measured/">elements_measured</a></td>
<td>Elements detected and measured</td>
</tr>
<tr>
<td><a href="elements/emission_filter/">emission_filter</a></td>
<td>Specifications of the emission filter</td>
</tr>
<tr>
<td><a href="elements/emission_wavelength/">emission_wavelength</a></td>
<td>Emission wavelength in nanometers</td>
</tr>
<tr>
<td><a href="elements/energy_max/">energy_max</a></td>
<td>Maximum X-ray energy in keV</td>
</tr>
<tr>
<td><a href="elements/energy_min/">energy_min</a></td>
<td>Minimum X-ray energy in keV</td>
</tr>
<tr>
<td><a href="elements/excitation_filter/">excitation_filter</a></td>
<td>Specifications of the excitation filter</td>
</tr>
<tr>
<td><a href="elements/excitation_wavelength/">excitation_wavelength</a></td>
<td>Excitation wavelength in nanometers</td>
</tr>
<tr>
<td><a href="elements/experiment_code/">experiment_code</a></td>
<td>Unique experiment identifier</td>
</tr>
<tr>
<td><a href="elements/experiment_date/">experiment_date</a></td>
<td>Date of the experiment</td>
</tr>
<tr>
<td><a href="elements/experiment_id/">experiment_id</a></td>
<td>Reference to the source experiment</td>
</tr>
<tr>
<td><a href="elements/experimental_conditions/">experimental_conditions</a></td>
<td>Environmental and experimental conditions</td>
</tr>
<tr>
<td><a href="elements/exposure_time/">exposure_time</a></td>
<td>Exposure time in seconds</td>
</tr>
<tr>
<td><a href="elements/file_format/">file_format</a></td>
<td>File format</td>
</tr>
<tr>
<td><a href="elements/file_name/">file_name</a></td>
<td>Name of the file</td>
</tr>
<tr>
<td><a href="elements/file_path/">file_path</a></td>
<td>Path to the file</td>
</tr>
<tr>
<td><a href="elements/file_size_bytes/">file_size_bytes</a></td>
<td>File size in bytes</td>
</tr>
<tr>
<td><a href="elements/flash_cooling_method/">flash_cooling_method</a></td>
<td>Flash cooling protocol</td>
</tr>
<tr>
<td><a href="elements/fluorophore/">fluorophore</a></td>
<td>Name or type of fluorophore used</td>
</tr>
<tr>
<td><a href="elements/flux/">flux</a></td>
<td>Photon flux in photons/second</td>
</tr>
<tr>
<td><a href="elements/flux_density/">flux_density</a></td>
<td>Photon flux density in photons/s/mm²</td>
</tr>
<tr>
<td><a href="elements/frame_rate/">frame_rate</a></td>
<td>Frames per second</td>
</tr>
<tr>
<td><a href="elements/goniometer_type/">goniometer_type</a></td>
<td>Type of goniometer</td>
</tr>
<tr>
<td><a href="elements/gpu_hours/">gpu_hours</a></td>
<td>GPU hours used</td>
</tr>
<tr>
<td><a href="elements/grid_type/">grid_type</a></td>
<td>Type of EM grid used</td>
</tr>
<tr>
<td><a href="elements/hole_size/">hole_size</a></td>
<td>Hole size in micrometers</td>
</tr>
<tr>
<td><a href="elements/humidity/">humidity</a></td>
<td>Humidity percentage</td>
</tr>
<tr>
<td><a href="elements/humidity_percentage/">humidity_percentage</a></td>
<td>Chamber humidity during vitrification</td>
</tr>
<tr>
<td><a href="elements/id/">id</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/illumination_type/">illumination_type</a></td>
<td>Type of illumination (brightfield, darkfield, phase contrast, DIC)</td>
</tr>
<tr>
<td><a href="elements/images/">images</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/installation_date/">installation_date</a></td>
<td>Date of instrument installation</td>
</tr>
<tr>
<td><a href="elements/instrument_code/">instrument_code</a></td>
<td>Unique identifier code for the instrument</td>
</tr>
<tr>
<td><a href="elements/instrument_id/">instrument_id</a></td>
<td>Reference to the instrument used</td>
</tr>
<tr>
<td><a href="elements/instrument_runs/">instrument_runs</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/keywords/">keywords</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/label/">label</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/laser_power/">laser_power</a></td>
<td>Laser power in milliwatts or percentage</td>
</tr>
<tr>
<td><a href="elements/ligands/">ligands</a></td>
<td>Bound ligands or cofactors</td>
</tr>
<tr>
<td><a href="elements/magnification/">magnification</a></td>
<td>Optical magnification factor</td>
</tr>
<tr>
<td><a href="elements/manufacturer/">manufacturer</a></td>
<td>Instrument manufacturer</td>
</tr>
<tr>
<td><a href="elements/memory_gb/">memory_gb</a></td>
<td>Maximum memory used in GB</td>
</tr>
<tr>
<td><a href="elements/model/">model</a></td>
<td>Instrument model</td>
</tr>
<tr>
<td><a href="elements/modifications/">modifications</a></td>
<td>Post-translational modifications or chemical modifications</td>
</tr>
<tr>
<td><a href="elements/molecular_composition/">molecular_composition</a></td>
<td>Description of molecular composition including sequences, modifications, liga...</td>
</tr>
<tr>
<td><a href="elements/molecular_signatures/">molecular_signatures</a></td>
<td>Identified molecular signatures or peaks</td>
</tr>
<tr>
<td><a href="elements/molecular_weight/">molecular_weight</a></td>
<td>Molecular weight in kDa</td>
</tr>
<tr>
<td><a href="elements/monochromator_type/">monochromator_type</a></td>
<td>Type of monochromator</td>
</tr>
<tr>
<td><a href="elements/mounting_method/">mounting_method</a></td>
<td>Crystal mounting method</td>
</tr>
<tr>
<td><a href="elements/number_of_scans/">number_of_scans</a></td>
<td>Number of scans averaged for the spectrum</td>
</tr>
<tr>
<td><a href="elements/numerical_aperture/">numerical_aperture</a></td>
<td>Numerical aperture of the objective lens</td>
</tr>
<tr>
<td><a href="elements/ontology/">ontology</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/operator_id/">operator_id</a></td>
<td>Person who performed the preparation</td>
</tr>
<tr>
<td><a href="elements/output_files/">output_files</a></td>
<td>Output files generated</td>
</tr>
<tr>
<td><a href="elements/parent_sample_id/">parent_sample_id</a></td>
<td>Reference to parent sample for derivation tracking</td>
</tr>
<tr>
<td><a href="elements/ph/">ph</a></td>
<td>pH of the buffer</td>
</tr>
<tr>
<td><a href="elements/phase_plate/">phase_plate</a></td>
<td>Phase plate available</td>
</tr>
<tr>
<td><a href="elements/pinhole_size/">pinhole_size</a></td>
<td>Pinhole size in Airy units for confocal microscopy</td>
</tr>
<tr>
<td><a href="elements/pixel_size/">pixel_size</a></td>
<td>Pixel size in Angstroms</td>
</tr>
<tr>
<td><a href="pixel_size_max.md">pixel_size_max</a></td>
<td>Maximum pixel size in Angstroms per pixel</td>
</tr>
<tr>
<td><a href="pixel_size_min.md">pixel_size_min</a></td>
<td>Minimum pixel size in Angstroms per pixel</td>
</tr>
<tr>
<td><a href="elements/plasma_treatment/">plasma_treatment</a></td>
<td>Plasma treatment details</td>
</tr>
<tr>
<td><a href="elements/preparation_date/">preparation_date</a></td>
<td>Date of sample preparation</td>
</tr>
<tr>
<td><a href="elements/preparation_method/">preparation_method</a></td>
<td>Method used to prepare the sample</td>
</tr>
<tr>
<td><a href="elements/preparation_type/">preparation_type</a></td>
<td>Type of sample preparation</td>
</tr>
<tr>
<td><a href="elements/pressure/">pressure</a></td>
<td>Pressure in kPa</td>
</tr>
<tr>
<td><a href="elements/processing_level/">processing_level</a></td>
<td>Processing level (0=raw, 1=corrected, 2=derived, 3=model)</td>
</tr>
<tr>
<td><a href="elements/processing_parameters/">processing_parameters</a></td>
<td>Parameters used in processing</td>
</tr>
<tr>
<td><a href="elements/processing_status/">processing_status</a></td>
<td>Current processing status</td>
</tr>
<tr>
<td><a href="elements/protocol_description/">protocol_description</a></td>
<td>Detailed protocol description</td>
</tr>
<tr>
<td><a href="elements/purity_percentage/">purity_percentage</a></td>
<td>Sample purity as percentage</td>
</tr>
<tr>
<td><a href="elements/q_range_max/">q_range_max</a></td>
<td>Maximum q value in inverse Angstroms</td>
</tr>
<tr>
<td><a href="elements/q_range_min/">q_range_min</a></td>
<td>Minimum q value in inverse Angstroms</td>
</tr>
<tr>
<td><a href="elements/quality_metrics/">quality_metrics</a></td>
<td>Quality control metrics for the sample</td>
</tr>
<tr>
<td><a href="elements/quantum_yield/">quantum_yield</a></td>
<td>Quantum yield of the fluorophore</td>
</tr>
<tr>
<td><a href="elements/r_factor/">r_factor</a></td>
<td>R-factor for crystallography</td>
</tr>
<tr>
<td><a href="elements/raw_data_location/">raw_data_location</a></td>
<td>Location of raw data files</td>
</tr>
<tr>
<td><a href="elements/reconstruction_method/">reconstruction_method</a></td>
<td>Method used for 3D reconstruction</td>
</tr>
<tr>
<td><a href="elements/resolution/">resolution</a></td>
<td>Resolution in Angstroms</td>
</tr>
<tr>
<td><a href="elements/sample_cell_type/">sample_cell_type</a></td>
<td>Type of sample cell used</td>
</tr>
<tr>
<td><a href="elements/sample_changer_capacity/">sample_changer_capacity</a></td>
<td>Number of samples in automatic sample changer</td>
</tr>
<tr>
<td><a href="elements/sample_code/">sample_code</a></td>
<td>Unique identifier code for the sample</td>
</tr>
<tr>
<td><a href="elements/sample_id/">sample_id</a></td>
<td>Reference to the sample being prepared</td>
</tr>
<tr>
<td><a href="elements/sample_preparations/">sample_preparations</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/sample_type/">sample_type</a></td>
<td>Type of biological sample</td>
</tr>
<tr>
<td><a href="elements/samples/">samples</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/sequences/">sequences</a></td>
<td>Amino acid or nucleotide sequences</td>
</tr>
<tr>
<td><a href="elements/signal_to_noise/">signal_to_noise</a></td>
<td>Signal to noise ratio</td>
</tr>
<tr>
<td><a href="elements/software_name/">software_name</a></td>
<td>Software used for processing</td>
</tr>
<tr>
<td><a href="elements/software_version/">software_version</a></td>
<td>Software version</td>
</tr>
<tr>
<td><a href="elements/source_type/">source_type</a></td>
<td>Type of X-ray source</td>
</tr>
<tr>
<td><a href="elements/spectral_resolution/">spectral_resolution</a></td>
<td>Spectral resolution in cm⁻¹</td>
</tr>
<tr>
<td><a href="elements/started_at/">started_at</a></td>
<td>Workflow start time</td>
</tr>
<tr>
<td><a href="elements/storage_conditions/">storage_conditions</a></td>
<td>Storage conditions for the sample</td>
</tr>
<tr>
<td><a href="elements/storage_gb/">storage_gb</a></td>
<td>Storage used in GB</td>
</tr>
<tr>
<td><a href="elements/studies/">studies</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/support_film/">support_film</a></td>
<td>Support film type</td>
</tr>
<tr>
<td><a href="elements/technique/">technique</a></td>
<td>Technique used for data collection</td>
</tr>
<tr>
<td><a href="elements/temperature/">temperature</a></td>
<td>Storage temperature in Celsius</td>
</tr>
<tr>
<td><a href="elements/temperature_control/">temperature_control</a></td>
<td>Temperature control settings</td>
</tr>
<tr>
<td><a href="elements/temperature_control_range/">temperature_control_range</a></td>
<td>Temperature control range in Celsius</td>
</tr>
<tr>
<td><a href="elements/temperature_unit/">temperature_unit</a></td>
<td>Temperature unit</td>
</tr>
<tr>
<td><a href="elements/terms/">terms</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/title/">title</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/total_dose/">total_dose</a></td>
<td>Total electron dose for cryo-EM</td>
</tr>
<tr>
<td><a href="elements/total_frames/">total_frames</a></td>
<td>Total number of frames/images</td>
</tr>
<tr>
<td><a href="elements/vitrification_method/">vitrification_method</a></td>
<td>Method used for vitrification</td>
</tr>
<tr>
<td><a href="elements/voxel_size/">voxel_size</a></td>
<td>Voxel size in Angstroms</td>
</tr>
<tr>
<td><a href="elements/wavenumber_max/">wavenumber_max</a></td>
<td>Maximum wavenumber in cm⁻¹</td>
</tr>
<tr>
<td><a href="elements/wavenumber_min/">wavenumber_min</a></td>
<td>Minimum wavenumber in cm⁻¹</td>
</tr>
<tr>
<td><a href="elements/white_balance/">white_balance</a></td>
<td>White balance settings</td>
</tr>
<tr>
<td><a href="elements/workflow_code/">workflow_code</a></td>
<td>Unique workflow identifier</td>
</tr>
<tr>
<td><a href="elements/workflow_runs/">workflow_runs</a></td>
<td></td>
</tr>
<tr>
<td><a href="elements/workflow_type/">workflow_type</a></td>
<td>Type of processing workflow</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="elements/CollectionModeEnum/">CollectionModeEnum</a></td>
<td>Data collection modes</td>
</tr>
<tr>
<td><a href="elements/ConcentrationUnitEnum/">ConcentrationUnitEnum</a></td>
<td>Units for concentration measurement</td>
</tr>
<tr>
<td><a href="elements/CrystallizationMethodEnum/">CrystallizationMethodEnum</a></td>
<td>Methods for protein crystallization</td>
</tr>
<tr>
<td><a href="elements/DataTypeEnum/">DataTypeEnum</a></td>
<td>Types of data</td>
</tr>
<tr>
<td><a href="elements/DetectorTypeEnum/">DetectorTypeEnum</a></td>
<td>Types of detectors for cryo-EM</td>
</tr>
<tr>
<td><a href="elements/FileFormatEnum/">FileFormatEnum</a></td>
<td>File formats</td>
</tr>
<tr>
<td><a href="elements/GridTypeEnum/">GridTypeEnum</a></td>
<td>Types of EM grids</td>
</tr>
<tr>
<td><a href="elements/IlluminationTypeEnum/">IlluminationTypeEnum</a></td>
<td>Types of illumination for optical microscopy</td>
</tr>
<tr>
<td><a href="elements/InstrumentStatusEnum/">InstrumentStatusEnum</a></td>
<td>Operational status of instruments</td>
</tr>
<tr>
<td><a href="elements/PreparationTypeEnum/">PreparationTypeEnum</a></td>
<td>Types of sample preparation</td>
</tr>
<tr>
<td><a href="elements/ProcessingStatusEnum/">ProcessingStatusEnum</a></td>
<td>Processing status</td>
</tr>
<tr>
<td><a href="elements/SampleTypeEnum/">SampleTypeEnum</a></td>
<td>Types of biological samples</td>
</tr>
<tr>
<td><a href="elements/TechniqueEnum/">TechniqueEnum</a></td>
<td>Structural biology techniques</td>
</tr>
<tr>
<td><a href="elements/TemperatureUnitEnum/">TemperatureUnitEnum</a></td>
<td>Units for temperature measurement</td>
</tr>
<tr>
<td><a href="elements/VitrificationMethodEnum/">VitrificationMethodEnum</a></td>
<td>Methods for vitrification</td>
</tr>
<tr>
<td><a href="elements/WorkflowTypeEnum/">WorkflowTypeEnum</a></td>
<td>Types of processing workflows</td>
</tr>
<tr>
<td><a href="elements/XRaySourceTypeEnum/">XRaySourceTypeEnum</a></td>
<td>Types of X-ray sources</td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="elements/Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="elements/Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="elements/Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="elements/DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="elements/Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="elements/Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="elements/Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="elements/Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="elements/Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="elements/Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="elements/Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="elements/Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="elements/Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="elements/Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="elements/Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="elements/String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="elements/Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="elements/Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="elements/Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="assets/javascripts/bundle.f55a23d4.min.js"></script>
      
    
  </body>
</html>