
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://lambda-ber.github.io/lambda-ber-schema/background/ome-alignment/">
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.22">
    
    
      
        <title>OME (Open Microscopy Environment) and lambda-ber-schema Alignment - lambda-ber-schema</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.84d31ad4.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="green">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#ome-open-microscopy-environment-and-lambda-ber-schema-alignment" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="lambda-ber-schema" class="md-header__button md-logo" aria-label="lambda-ber-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            lambda-ber-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              OME (Open Microscopy Environment) and lambda-ber-schema Alignment
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="green"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="amber" data-md-color-accent="green"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/lambda-ber/lambda-ber-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="lambda-ber-schema" class="md-nav__button md-logo" aria-label="lambda-ber-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    lambda-ber-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/lambda-ber/lambda-ber-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Index
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Background
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Background
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../nexus/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    NEXUS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../mmcif/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    mmCIF
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../empiar/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    EMPIAR
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../emdb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    EMDB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../dials/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    DIALS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../sasdb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SASDB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../rembi-alignment/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    REMBI
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../lambda-ber-schema-data-lakehouse-strategy/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Lakehouse Strategy
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ber-doe-facilities/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Integration with BER
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../data-quality-challenges/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Quality Challenges
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Examples
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Examples
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../examples/simplescattering-glurs-analysis/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Simple Scattering GluRS Dataset
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Technical
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Technical
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../technical/ome-mapping/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mapping to OME
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#executive-summary" class="md-nav__link">
    <span class="md-ellipsis">
      Executive Summary
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ome-overview" class="md-nav__link">
    <span class="md-ellipsis">
      OME Overview
    </span>
  </a>
  
    <nav class="md-nav" aria-label="OME Overview">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#what-is-ome" class="md-nav__link">
    <span class="md-ellipsis">
      What is OME?
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#core-components" class="md-nav__link">
    <span class="md-ellipsis">
      Core Components
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ome-data-model-structure" class="md-nav__link">
    <span class="md-ellipsis">
      OME Data Model Structure
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#lambda-ber-schema-ome-alignment" class="md-nav__link">
    <span class="md-ellipsis">
      lambda-ber-schema-OME Alignment
    </span>
  </a>
  
    <nav class="md-nav" aria-label="lambda-ber-schema-OME Alignment">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#shared-concepts" class="md-nav__link">
    <span class="md-ellipsis">
      Shared Concepts
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Shared Concepts">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-image-centric-data-model" class="md-nav__link">
    <span class="md-ellipsis">
      1. Image-Centric Data Model
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-instrument-metadata" class="md-nav__link">
    <span class="md-ellipsis">
      2. Instrument Metadata
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-experimental-context" class="md-nav__link">
    <span class="md-ellipsis">
      3. Experimental Context
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#complementary-extensions" class="md-nav__link">
    <span class="md-ellipsis">
      Complementary Extensions
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Complementary Extensions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-multi-technique-integration" class="md-nav__link">
    <span class="md-ellipsis">
      1. Multi-Technique Integration
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-molecular-composition" class="md-nav__link">
    <span class="md-ellipsis">
      2. Molecular Composition
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-processing-workflows" class="md-nav__link">
    <span class="md-ellipsis">
      3. Processing Workflows
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#interoperability-strategies" class="md-nav__link">
    <span class="md-ellipsis">
      Interoperability Strategies
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Interoperability Strategies">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-direct-mapping-for-shared-concepts" class="md-nav__link">
    <span class="md-ellipsis">
      1. Direct Mapping for Shared Concepts
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-ome-compatible-subclasses" class="md-nav__link">
    <span class="md-ellipsis">
      2. OME-Compatible Subclasses
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-bio-formats-integration-points" class="md-nav__link">
    <span class="md-ellipsis">
      3. Bio-Formats Integration Points
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4-omero-compatibility" class="md-nav__link">
    <span class="md-ellipsis">
      4. OMERO Compatibility
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#practical-integration-examples" class="md-nav__link">
    <span class="md-ellipsis">
      Practical Integration Examples
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Practical Integration Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-1-correlative-microscopy" class="md-nav__link">
    <span class="md-ellipsis">
      Example 1: Correlative Microscopy
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-2-multi-scale-imaging" class="md-nav__link">
    <span class="md-ellipsis">
      Example 2: Multi-Scale Imaging
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#benefits-of-alignment" class="md-nav__link">
    <span class="md-ellipsis">
      Benefits of Alignment
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Benefits of Alignment">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#for-users" class="md-nav__link">
    <span class="md-ellipsis">
      For Users
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#for-developers" class="md-nav__link">
    <span class="md-ellipsis">
      For Developers
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#for-facilities" class="md-nav__link">
    <span class="md-ellipsis">
      For Facilities
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#future-convergence-opportunities" class="md-nav__link">
    <span class="md-ellipsis">
      Future Convergence Opportunities
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Future Convergence Opportunities">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-ome-ngff-for-structural-biology" class="md-nav__link">
    <span class="md-ellipsis">
      1. OME-NGFF for Structural Biology
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-shared-ontologies" class="md-nav__link">
    <span class="md-ellipsis">
      2. Shared Ontologies
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#3-analysis-pipeline-integration" class="md-nav__link">
    <span class="md-ellipsis">
      3. Analysis Pipeline Integration
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4-unified-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      4. Unified Visualization
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#implementation-recommendations" class="md-nav__link">
    <span class="md-ellipsis">
      Implementation Recommendations
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Implementation Recommendations">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#phase-1-core-compatibility" class="md-nav__link">
    <span class="md-ellipsis">
      Phase 1: Core Compatibility
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phase-2-enhanced-integration" class="md-nav__link">
    <span class="md-ellipsis">
      Phase 2: Enhanced Integration
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#phase-3-full-convergence" class="md-nav__link">
    <span class="md-ellipsis">
      Phase 3: Full Convergence
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#conclusion" class="md-nav__link">
    <span class="md-ellipsis">
      Conclusion
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#references" class="md-nav__link">
    <span class="md-ellipsis">
      References
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="ome-open-microscopy-environment-and-lambda-ber-schema-alignment">OME (Open Microscopy Environment) and lambda-ber-schema Alignment</h1>
<h2 id="executive-summary">Executive Summary</h2>
<p>The Open Microscopy Environment (OME) provides established standards for biological imaging data, while lambda-ber-schema extends these concepts to structural biology techniques. This document outlines how lambda-ber-schema aligns with and complements OME standards, enabling interoperability between light/fluorescence microscopy and structural biology data.</p>
<h2 id="ome-overview">OME Overview</h2>
<h3 id="what-is-ome">What is OME?</h3>
<p>The Open Microscopy Environment is a consortium developing open-source software and format standards for biological microscopy data. Established in 2005, OME has become the de facto standard for microscopy data management in the life sciences.</p>
<h3 id="core-components">Core Components</h3>
<ol>
<li><strong>OME Data Model</strong>: A comprehensive specification for storing biological imaging data</li>
<li><strong>Bio-Formats</strong>: Library translating 160+ proprietary formats into the common OME model</li>
<li><strong>OMERO</strong>: Server platform for managing, visualizing, and analyzing microscopy data</li>
<li><strong>OME-TIFF</strong>: Standard file format combining TIFF images with OME-XML metadata</li>
<li><strong>OME-NGFF</strong>: Next-generation cloud-native formats (Zarr-based)</li>
</ol>
<h3 id="ome-data-model-structure">OME Data Model Structure</h3>
<p>The OME model centers around these key entities:</p>
<pre><code>Image
├── Pixels (dimensional and type information)
│   ├── Channel (wavelength, fluorophore)
│   ├── Plane (2D slice with timing)
│   └── BinData/TiffData (raw pixel data)
├── Instrument (microscope configuration)
│   ├── Microscope
│   ├── Objective
│   ├── Detector
│   └── LightSource
├── Experiment (experimental context)
├── Experimenter (person metadata)
└── ROI (Regions of Interest)
    └── Shape (rectangles, polygons, etc.)
</code></pre>
<h2 id="lambda-ber-schema-ome-alignment">lambda-ber-schema-OME Alignment</h2>
<h3 id="shared-concepts">Shared Concepts</h3>
<p>lambda-ber-schema intentionally aligns with OME in several fundamental areas:</p>
<h4 id="1-image-centric-data-model">1. Image-Centric Data Model</h4>
<p><strong>OME Approach:</strong>
- Images are first-class entities with rich metadata
- Hierarchical structure from Image → Pixels → Planes</p>
<p><strong>lambda-ber-schema Alignment:</strong></p>
<pre><code class="language-yaml">Image (abstract base)
├── Image2D (micrographs, diffraction patterns)
├── Image3D (volumes, tomograms)
├── FluorescenceImage (OME-compatible fluorescence)
├── OpticalImage (brightfield, phase contrast)
├── FTIRImage (spectroscopy imaging)
└── XRFImage (elemental mapping)
</code></pre>
<h4 id="2-instrument-metadata">2. Instrument Metadata</h4>
<p><strong>OME Model:</strong></p>
<pre><code class="language-xml">&lt;Instrument ID=&quot;Instrument:1&quot;&gt;
  &lt;Microscope Type=&quot;Inverted&quot;/&gt;
  &lt;Objective ID=&quot;Objective:1&quot; LensNA=&quot;1.4&quot;/&gt;
  &lt;Detector Type=&quot;PMT&quot;/&gt;
&lt;/Instrument&gt;
</code></pre>
<p><strong>lambda-ber-schema Parallel:</strong></p>
<pre><code class="language-yaml">Instrument (abstract base)
├── CryoEMInstrument
│   ├── voltage: 300kV
│   ├── detector_type: direct_electron
│   └── pixel_size: 0.83Å
├── XRayInstrument
│   ├── source_type: synchrotron
│   └── wavelength: 0.97Å
└── SAXSInstrument
    ├── q_range: 0.01-4.0 Å⁻¹
    └── detector_type: hybrid_pixel
</code></pre>
<h4 id="3-experimental-context">3. Experimental Context</h4>
<p>Both models capture:
- Experimenter information
- Acquisition parameters
- Sample preparation details
- Processing workflows</p>
<h3 id="complementary-extensions">Complementary Extensions</h3>
<p>lambda-ber-schema extends OME concepts for structural biology:</p>
<h4 id="1-multi-technique-integration">1. Multi-Technique Integration</h4>
<p>While OME focuses on light microscopy, lambda-ber-schema adds:
- Cryo-EM specific parameters (defocus, CTF, particle picking)
- X-ray crystallography (space groups, unit cells, phasing)
- Small-angle scattering (Guinier analysis, P(r) distributions)
- Spectroscopy imaging (FTIR, XRF)</p>
<h4 id="2-molecular-composition">2. Molecular Composition</h4>
<p>lambda-ber-schema adds detailed biomolecular information:</p>
<pre><code class="language-yaml">MolecularComposition:
  sequences: [amino acid/nucleotide sequences]
  modifications: [PTMs, chemical modifications]
  ligands: [bound cofactors, substrates]
</code></pre>
<p>OME typically handles this through structured annotations.</p>
<h4 id="3-processing-workflows">3. Processing Workflows</h4>
<p>lambda-ber-schema explicitly models computational workflows:</p>
<pre><code class="language-yaml">WorkflowRun:
  workflow_type: refinement
  software_name: RELION
  processing_level: 3  # raw→processed→refined
</code></pre>
<p>OME uses AnalysisModule for similar purposes but with less domain specificity.</p>
<h2 id="interoperability-strategies">Interoperability Strategies</h2>
<h3 id="1-direct-mapping-for-shared-concepts">1. Direct Mapping for Shared Concepts</h3>
<table>
<thead>
<tr>
<th>OME Entity</th>
<th>lambda-ber-schema Entity</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>Image</td>
<td>Image</td>
<td>Base class alignment</td>
</tr>
<tr>
<td>Pixels</td>
<td>Image2D/3D dimensions</td>
<td>Dimensional metadata</td>
</tr>
<tr>
<td>Channel</td>
<td>FluorescenceImage.excitation_wavelength</td>
<td>Spectral information</td>
</tr>
<tr>
<td>Instrument</td>
<td>Instrument</td>
<td>Polymorphic specialization</td>
</tr>
<tr>
<td>Experimenter</td>
<td>Study.principal_investigator</td>
<td>Personnel tracking</td>
</tr>
<tr>
<td>ROI</td>
<td>Not yet implemented</td>
<td>Future enhancement</td>
</tr>
</tbody>
</table>
<h3 id="2-ome-compatible-subclasses">2. OME-Compatible Subclasses</h3>
<p>lambda-ber-schema's <code>FluorescenceImage</code> and <code>OpticalImage</code> classes are designed to be OME-compatible:</p>
<pre><code class="language-yaml">FluorescenceImage:
  excitation_wavelength: 488  # nm, maps to OME Channel
  emission_wavelength: 520    # nm
  fluorophore: &quot;GFP&quot;          # OME Fluor
  exposure_time: 100           # ms, OME Plane.ExposureTime
</code></pre>
<h3 id="3-bio-formats-integration-points">3. Bio-Formats Integration Points</h3>
<p>lambda-ber-schema data can interface with Bio-Formats through:
- Standard image formats (TIFF, PNG, JPEG)
- OME-TIFF export for compatible image types
- Metadata translation layers</p>
<h3 id="4-omero-compatibility">4. OMERO Compatibility</h3>
<p>Structural biology images can be stored in OMERO with:
- Custom annotations for technique-specific metadata
- Tag-based organization for multi-technique studies
- Integration with existing OMERO analysis workflows</p>
<h2 id="practical-integration-examples">Practical Integration Examples</h2>
<h3 id="example-1-correlative-microscopy">Example 1: Correlative Microscopy</h3>
<p>Study combining fluorescence microscopy (OME) with cryo-EM (lambda-ber-schema):</p>
<pre><code class="language-yaml">Study:
  images:
    - id: fluor-001
      _type: FluorescenceImage  # OME-compatible
      file_name: &quot;cells_gfp.ome.tiff&quot;
      excitation_wavelength: 488
      emission_wavelength: 520

    - id: cryoem-001
      _type: Image2D  # lambda-ber-schema cryo-EM
      file_name: &quot;micrograph_001.mrc&quot;
      defocus: -2.5
      pixel_size: 1.06
</code></pre>
<h3 id="example-2-multi-scale-imaging">Example 2: Multi-Scale Imaging</h3>
<p>Linking OME light microscopy with lambda-ber-schema X-ray imaging:</p>
<pre><code class="language-yaml">images:
  - id: optical-overview
    _type: OpticalImage
    illumination_type: brightfield
    magnification: 10
    # Compatible with OME Objective.NominalMagnification

  - id: xrf-map
    _type: XRFImage  
    beam_energy: 12.4  # keV
    beam_size: 2.0     # μm
    # lambda-ber-schema-specific X-ray fluorescence
</code></pre>
<h2 id="benefits-of-alignment">Benefits of Alignment</h2>
<h3 id="for-users">For Users</h3>
<ol>
<li><strong>Familiar concepts</strong>: Users experienced with OME find similar patterns in lambda-ber-schema</li>
<li><strong>Tool reuse</strong>: Existing OME tools can process compatible lambda-ber-schema data</li>
<li><strong>Unified workflows</strong>: Correlative studies span both frameworks seamlessly</li>
</ol>
<h3 id="for-developers">For Developers</h3>
<ol>
<li><strong>Established patterns</strong>: Leverage proven OME design patterns</li>
<li><strong>Interoperability</strong>: Data exchange between OME and lambda-ber-schema systems</li>
<li><strong>Community standards</strong>: Build on 15+ years of OME community experience</li>
</ol>
<h3 id="for-facilities">For Facilities</h3>
<ol>
<li><strong>Integrated management</strong>: Single platform for light and structural biology</li>
<li><strong>Metadata consistency</strong>: Unified approach across techniques</li>
<li><strong>Compliance</strong>: Meet FAIR data principles through established standards</li>
</ol>
<h2 id="future-convergence-opportunities">Future Convergence Opportunities</h2>
<h3 id="1-ome-ngff-for-structural-biology">1. OME-NGFF for Structural Biology</h3>
<p>Extend OME's Next-Generation File Format (Zarr-based) for:
- Large cryo-EM datasets
- Multi-resolution electron tomography
- Cloud-native structural biology workflows</p>
<h3 id="2-shared-ontologies">2. Shared Ontologies</h3>
<p>Develop common controlled vocabularies:
- Sample types (cells, proteins, complexes)
- Preparation methods
- Quality metrics</p>
<h3 id="3-analysis-pipeline-integration">3. Analysis Pipeline Integration</h3>
<p>Create adapters between:
- OMERO.scripts ↔ lambda-ber-schema WorkflowRuns
- Bio-Formats readers ↔ Structural biology formats
- ImageJ/Fiji plugins ↔ Structural biology tools</p>
<h3 id="4-unified-visualization">4. Unified Visualization</h3>
<p>Develop viewers supporting both:
- OME pyramidal TIFF for large 2D images
- lambda-ber-schema MRC/MAP files for 3D densities
- Correlative overlay capabilities</p>
<h2 id="implementation-recommendations">Implementation Recommendations</h2>
<h3 id="phase-1-core-compatibility">Phase 1: Core Compatibility</h3>
<ul>
<li>Ensure base Image class alignment</li>
<li>Implement OME-TIFF export for applicable images</li>
<li>Create Bio-Formats metadata translators</li>
</ul>
<h3 id="phase-2-enhanced-integration">Phase 2: Enhanced Integration</h3>
<ul>
<li>Develop OMERO.table definitions for lambda-ber-schema metadata</li>
<li>Create Bio-Formats readers for common structural formats</li>
<li>Implement OME-XML serialization for compatible entities</li>
</ul>
<h3 id="phase-3-full-convergence">Phase 3: Full Convergence</h3>
<ul>
<li>Contribute structural biology extensions to OME model</li>
<li>Develop unified OME-NGFF specification</li>
<li>Create reference implementation and validators</li>
</ul>
<h2 id="conclusion">Conclusion</h2>
<p>lambda-ber-schema's alignment with OME standards creates a bridge between traditional microscopy and structural biology data management. This alignment enables:</p>
<ol>
<li><strong>Reuse</strong> of established tools and workflows</li>
<li><strong>Interoperability</strong> between imaging modalities</li>
<li><strong>Unified</strong> data management strategies</li>
<li><strong>Community</strong> convergence around common standards</li>
</ol>
<p>By building on OME's successful foundation while extending it for structural biology's unique needs, lambda-ber-schema facilitates the integration of multi-scale, multi-modal biological imaging data essential for modern integrative structural biology.</p>
<h2 id="references">References</h2>
<ul>
<li>OME Data Model Specification: https://docs.openmicroscopy.org/ome-model/</li>
<li>Bio-Formats Documentation: https://docs.openmicroscopy.org/bio-formats/</li>
<li>OMERO Platform: https://www.openmicroscopy.org/omero/</li>
<li>OME-NGFF Specification: https://ngff.openmicroscopy.org/</li>
<li>Goldberg et al. (2005) "The Open Microscopy Environment (OME) Data Model and XML file" Genome Biology 6:R47</li>
</ul>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["content.tabs.link"], "search": "../../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
    
  </body>
</html>