
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://lambda-ber.github.io/lambda-ber-schema/slides/ontologies-for-structural-biology/">
      
      
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.22">
    
    
      
        <title>Ontologies for Structural Biology Imaging - lambda-ber-schema</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.84d31ad4.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="green">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#ontologies-for-structural-biology-imaging" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="lambda-ber-schema" class="md-header__button md-logo" aria-label="lambda-ber-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            lambda-ber-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Ontologies for Structural Biology Imaging
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="green"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="amber" data-md-color-accent="green"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/lambda-ber/lambda-ber-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="lambda-ber-schema" class="md-nav__button md-logo" aria-label="lambda-ber-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    lambda-ber-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/lambda-ber/lambda-ber-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Index
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Background
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Background
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/nexus/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    NEXUS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/mmcif/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    mmCIF
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/empiar/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    EMPIAR
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/emdb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    EMDB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/dials/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    DIALS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/sasdb/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SASDB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/rembi-alignment/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    REMBI
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/lambda-ber-schema-data-lakehouse-strategy/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Lakehouse Strategy
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/ber-doe-facilities/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Integration with BER
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../background/data-quality-challenges/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Quality Challenges
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Examples
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Examples
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../examples/simplescattering-glurs-analysis/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Simple Scattering GluRS Dataset
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Technical
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Technical
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../technical/ome-mapping/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mapping to OME
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Slides
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Slides
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../overview.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../technical-overview.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Technical Deep Dive
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#overview" class="md-nav__link">
    <span class="md-ellipsis">
      Overview
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chmo-chemical-methods-ontology" class="md-nav__link">
    <span class="md-ellipsis">
      CHMO: Chemical Methods Ontology
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chmo-structural-biology-coverage" class="md-nav__link">
    <span class="md-ellipsis">
      CHMO: Structural Biology Coverage
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#chmo-use-in-lambda-ber-schema" class="md-nav__link">
    <span class="md-ellipsis">
      CHMO: Use in lambda-ber-schema
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-biological-imaging-methods-ontology" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: Biological Imaging Methods Ontology
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-9-primary-axes" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: 9 Primary Axes
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-sample-preparation-coverage" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: Sample Preparation Coverage
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-visualization-methods" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: Visualization Methods
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-imaging-methods" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: Imaging Methods
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-illumination-detection" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: Illumination &amp; Detection
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#fbbi-gaps-for-structural-biology" class="md-nav__link">
    <span class="md-ellipsis">
      FBbi: Gaps for Structural Biology
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obi-ontology-for-biomedical-investigations" class="md-nav__link">
    <span class="md-ellipsis">
      OBI: Ontology for Biomedical Investigations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obi-core-structure" class="md-nav__link">
    <span class="md-ellipsis">
      OBI: Core Structure
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obi-structural-biology-assays" class="md-nav__link">
    <span class="md-ellipsis">
      OBI: Structural Biology Assays
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obi-specialized-assays" class="md-nav__link">
    <span class="md-ellipsis">
      OBI: Specialized Assays
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obi-devices-instruments" class="md-nav__link">
    <span class="md-ellipsis">
      OBI: Devices &amp; Instruments
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#obi-sample-preparation" class="md-nav__link">
    <span class="md-ellipsis">
      OBI: Sample Preparation
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ror-research-organization-registry" class="md-nav__link">
    <span class="md-ellipsis">
      ROR: Research Organization Registry
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ror-use-in-lambda-ber-schema" class="md-nav__link">
    <span class="md-ellipsis">
      ROR: Use in lambda-ber-schema
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#ror-benefits" class="md-nav__link">
    <span class="md-ellipsis">
      ROR: Benefits
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#coverage-comparison-matrix" class="md-nav__link">
    <span class="md-ellipsis">
      Coverage Comparison Matrix
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#strengths-by-ontology" class="md-nav__link">
    <span class="md-ellipsis">
      Strengths by Ontology
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gaps-limitations" class="md-nav__link">
    <span class="md-ellipsis">
      Gaps &amp; Limitations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#current-usage-in-lambda-ber-schema" class="md-nav__link">
    <span class="md-ellipsis">
      Current Usage in lambda-ber-schema
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#recommendations" class="md-nav__link">
    <span class="md-ellipsis">
      Recommendations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#future-enhancements" class="md-nav__link">
    <span class="md-ellipsis">
      Future Enhancements
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#integration-strategy" class="md-nav__link">
    <span class="md-ellipsis">
      Integration Strategy
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#best-practices" class="md-nav__link">
    <span class="md-ellipsis">
      Best Practices
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#resources" class="md-nav__link">
    <span class="md-ellipsis">
      Resources
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#summary" class="md-nav__link">
    <span class="md-ellipsis">
      Summary
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#questions" class="md-nav__link">
    <span class="md-ellipsis">
      Questions?
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="ontologies-for-structural-biology-imaging">Ontologies for Structural Biology Imaging</h1>
<p>A Survey of Relevant Ontologies and Their Coverage for lambda-ber-schema</p>
<hr />
<h2 id="overview">Overview</h2>
<p>This presentation surveys ontologies relevant to structural biology and bioimaging:</p>
<ol>
<li><strong>CHMO</strong> - Chemical Methods Ontology</li>
<li><strong>FBbi</strong> - Biological Imaging Methods Ontology</li>
<li><strong>OBI</strong> - Ontology for Biomedical Investigations</li>
<li><strong>ROR</strong> - Research Organization Registry</li>
<li>Coverage comparison and recommendations</li>
</ol>
<hr />
<h2 id="chmo-chemical-methods-ontology">CHMO: Chemical Methods Ontology</h2>
<p><strong>Focus</strong>: Analytical chemistry and experimental methods</p>
<p><strong>Scope</strong>:
- Chemical analysis techniques
- Sample preparation methods
- Instrumentation
- Data collection methods</p>
<p><strong>Maintainer</strong>: Royal Society of Chemistry (RSC)
<strong>Status</strong>: Active (OBO Foundry)</p>
<hr />
<h2 id="chmo-structural-biology-coverage">CHMO: Structural Biology Coverage</h2>
<p><strong>Excellent coverage</strong> for core techniques:</p>
<table>
<thead>
<tr>
<th>Technique</th>
<th>CHMO ID</th>
<th>Term</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cryo-EM</td>
<td>CHMO:0002413</td>
<td>cryogenic electron microscopy</td>
</tr>
<tr>
<td>X-ray crystallography</td>
<td>CHMO:0000156</td>
<td>X-ray diffraction</td>
</tr>
<tr>
<td>SAXS</td>
<td>CHMO:0000204</td>
<td>small-angle X-ray scattering</td>
</tr>
<tr>
<td>WAXS</td>
<td>CHMO:0000207</td>
<td>wide-angle X-ray scattering</td>
</tr>
<tr>
<td>SANS</td>
<td>CHMO:0000184</td>
<td>small-angle neutron scattering</td>
</tr>
<tr>
<td>Electron microscopy</td>
<td>CHMO:0000068</td>
<td>electron microscopy</td>
</tr>
<tr>
<td>Mass spectrometry</td>
<td>CHMO:0000470</td>
<td>mass spectrometry</td>
</tr>
</tbody>
</table>
<hr />
<h2 id="chmo-use-in-lambda-ber-schema">CHMO: Use in lambda-ber-schema</h2>
<p><strong>Currently implemented</strong> for TechniqueEnum and ExperimentalMethodEnum:</p>
<pre><code class="language-yaml">TechniqueEnum:
  cryo_em:
    description: &quot;Cryo-electron microscopy&quot;
    meaning: CHMO:0002413
  xray_crystallography:
    description: &quot;X-ray crystallography&quot;
    meaning: CHMO:0000156
  saxs:
    description: &quot;Small-angle X-ray scattering&quot;
    meaning: CHMO:0000204
</code></pre>
<p>‚úÖ <strong>Best choice for analytical techniques</strong></p>
<hr />
<h2 id="fbbi-biological-imaging-methods-ontology">FBbi: Biological Imaging Methods Ontology</h2>
<p><strong>Focus</strong>: Microscopy and biological imaging workflows</p>
<p><strong>Scope</strong>: Multi-dimensional imaging metadata covering:
- Sample preparation
- Visualization methods (probes, dyes, FPs)
- Illumination methods
- Detection methods
- Imaged parameters
- Resolution enhancement</p>
<p><strong>Maintainer</strong>: Originally FlyBase, now community
<strong>Status</strong>: Inactive (last update 2020)</p>
<hr />
<h2 id="fbbi-9-primary-axes">FBbi: 9 Primary Axes</h2>
<ol>
<li><strong>Sample Preparation</strong> - fixation, embedding, sectioning</li>
<li><strong>Visualization Method</strong> - fluorescent proteins, dyes, stains</li>
<li><strong>Imaging Method</strong> - microscopy types</li>
<li><strong>Illumination Method</strong> - photons, electrons, X-rays</li>
<li><strong>Detection Method</strong> - CCD, CMOS, PMT, film</li>
<li><strong>Imaged Parameter</strong> - fluorescence, scattering, absorption</li>
<li><strong>Contrast Enhancement</strong> - methods to improve contrast</li>
<li><strong>Resolution Enhancement</strong> - super-resolution techniques</li>
<li><strong>Source of Contrast</strong> - what creates image contrast</li>
</ol>
<hr />
<h2 id="fbbi-sample-preparation-coverage">FBbi: Sample Preparation Coverage</h2>
<p><strong>Comprehensive fixation methods</strong>:
- Chemical: formaldehyde, glutaraldehyde, osmium tetroxide
- Cryofixation: FBbi:00000013
- Freeze-substitution: FBbi:00000621</p>
<p><strong>Embedding</strong>:
- Vitreous ice: FBbi:00000089
- Epoxy resin, wax, paraffin</p>
<p><strong>Sectioning</strong>:
- Microtome, cryostat, vibratome</p>
<p><strong>State</strong>: living, fixed, permeabilized</p>
<hr />
<h2 id="fbbi-visualization-methods">FBbi: Visualization Methods</h2>
<p><strong>Extensive probe coverage</strong>:
- <strong>100+ fluorescent proteins</strong>: GFP variants, RFP, mCherry, tdTomato
- <strong>Chemical dyes</strong>: DAPI, propidium iodide, Alexa Fluors
- <strong>Lipid probes</strong>: DiI, DiA
- <strong>Nucleic acid probes</strong>
- <strong>Electron-dense stains</strong>
- <strong>Radioisotopes</strong></p>
<p><strong>Most comprehensive for microscopy probes</strong></p>
<hr />
<h2 id="fbbi-imaging-methods">FBbi: Imaging Methods</h2>
<p><strong>Microscopy types</strong>:
- Electron microscopy: FBbi:00000256
  - TEM: FBbi:00000258
  - Scanning-transmission EM: FBbi:00000380
  - High-voltage EM: FBbi:00000622
- Light microscopy: FBbi:00000345
  - Confocal: FBbi:00000251
  - Two-photon: FBbi:00000254
  - Fluorescence: FBbi:00000246
- X-ray microscopy: FBbi:00000260</p>
<hr />
<h2 id="fbbi-illumination-detection">FBbi: Illumination &amp; Detection</h2>
<p><strong>Illumination sources</strong>:
- Photons: UV/visible/IR (FBbi:00000341)
- Electrons: FBbi:00000273
- X-rays: FBbi:00000342
- Neutrons: FBbi:00000275
- Ions: FBbi:00000276</p>
<p><strong>Detection methods</strong>:
- CCD, EMCCD, CMOS: FBbi:00000294, 00000301, 00000304
- PMT: FBbi:00000295
- Film: FBbi:00000303
- X-ray photon detection: FBbi:00000379</p>
<hr />
<h2 id="fbbi-gaps-for-structural-biology">FBbi: Gaps for Structural Biology</h2>
<p><strong>Missing or limited</strong>:
- ‚ùå No specific cryo-EM term (only general EM)
- ‚ùå No X-ray crystallography
- ‚ùå No SAXS/SANS terms
- ‚ùå No synchrotron-specific terms
- ‚ùå Limited scattering terminology</p>
<p><strong>Best for</strong>: Cellular/subcellular microscopy workflows
<strong>Less suitable for</strong>: Molecular structure determination</p>
<hr />
<h2 id="obi-ontology-for-biomedical-investigations">OBI: Ontology for Biomedical Investigations</h2>
<p><strong>Focus</strong>: Entire investigation lifecycle</p>
<p><strong>Scope</strong>:
- Planning and design
- Execution (assays, protocols)
- Materials and reagents
- Instruments and devices
- Data generation and analysis
- Roles and participants</p>
<p><strong>Maintainer</strong>: OBI Consortium
<strong>Status</strong>: Active (OBO Foundry), 2500+ terms</p>
<hr />
<h2 id="obi-core-structure">OBI: Core Structure</h2>
<p><strong>Top-level concepts</strong>:
- <strong>Assay</strong> (OBI:0000070) - experimental procedures
- <strong>Protocol</strong> (OBI:0000272) - planned processes
- <strong>Device</strong> (OBI:0000968) - instruments
- <strong>Material entity</strong> - samples, reagents
- <strong>Data item</strong> - results, measurements
- <strong>Study design</strong> - experimental planning
- <strong>Role</strong> - investigator, manufacturer</p>
<p><strong>Integration philosophy</strong>: Reuses existing ontologies via MIREOT</p>
<hr />
<h2 id="obi-structural-biology-assays">OBI: Structural Biology Assays</h2>
<p><strong>3D Structure Determination</strong>:
- X-ray crystallography: OBI:0000912
- Electron microscopy: OBI:0003118
- NMR: OBI:0000182
- Small-angle scattering: OBI:0002108</p>
<p><strong>Imaging assays</strong>:
- Microscopy assay: OBI:0002119
- Electron microscopy imaging: OBI:0001631
- Widefield microscopy: OBI:0002437
- Lightsheet fluorescence: OBI:0003098</p>
<hr />
<h2 id="obi-specialized-assays">OBI: Specialized Assays</h2>
<p><strong>X-ray crystallography variants</strong>:
- MHC:ligand complex: OBI:0001595
- T cell epitope:MHC:TCR: OBI:0001311
- B cell epitope:antibody: OBI:0001738</p>
<p><strong>Electron microscopy variants</strong>:
- B cell epitope:antibody: OBI:0001646
- T cell epitope:MHC:TCR: OBI:0003612
- MHC:ligand complex: OBI:0003613</p>
<p><strong>Focus on immunology applications</strong></p>
<hr />
<h2 id="obi-devices-instruments">OBI: Devices &amp; Instruments</h2>
<p><strong>Device hierarchy</strong>:
- Measurement device: OBI:0000832
- Image creation device: OBI:0000398
- Material separation device: OBI:0000932
- Environmental control: OBI:0001034
- Cryofixation device: OBI:0001074</p>
<p><strong>More abstract than equipment catalogs</strong>
<strong>Good for describing roles, not specific models</strong></p>
<hr />
<h2 id="obi-sample-preparation">OBI: Sample Preparation</h2>
<p><strong>Integration approach</strong>:
- OBI references sample prep terms
- Imports from CHMO for chemical methods
- Imports from other ontologies via MIREOT</p>
<p><strong>Example</strong>:
- Sample preparation for assay: OBI:0000073
- Specimen collection protocol: OBI:0003070</p>
<p><strong>Coordinates rather than duplicates</strong></p>
<hr />
<h2 id="ror-research-organization-registry">ROR: Research Organization Registry</h2>
<p><strong>Focus</strong>: Persistent organizational identifiers</p>
<p><strong>Scope</strong>:
- Research institutions
- Funding organizations
- Facilities
- Universities and labs</p>
<p><strong>Maintainer</strong>: Community initiative
<strong>Status</strong>: Active, constantly updated</p>
<hr />
<h2 id="ror-use-in-lambda-ber-schema">ROR: Use in lambda-ber-schema</h2>
<p><strong>FacilityEnum with ROR mappings</strong>:</p>
<pre><code class="language-yaml">NSLS_II:
  title: &quot;National Synchrotron Light Source II&quot;
  meaning: ROR:01q47ea17
  annotations:
    parent_ror: &quot;ROR:02ex6cf31&quot;  # Brookhaven
    country: &quot;USA&quot;
    doe_facility: &quot;true&quot;

ESRF:
  title: &quot;European Synchrotron Radiation Facility&quot;
  meaning: ROR:02550n020
  annotations:
    country: &quot;France&quot;
</code></pre>
<hr />
<h2 id="ror-benefits">ROR: Benefits</h2>
<p>‚úÖ <strong>Persistent identifiers</strong> for facilities
‚úÖ <strong>Global coverage</strong> of research organizations
‚úÖ <strong>Hierarchical relationships</strong> (facility ‚Üí parent lab)
‚úÖ <strong>Regular updates</strong> with new facilities
‚úÖ <strong>API access</strong> for programmatic lookup
‚úÖ <strong>Cross-referencing</strong> with Wikidata, GRID, ISNI</p>
<p><strong>Ideal for facility/organization identification</strong></p>
<hr />
<h2 id="coverage-comparison-matrix">Coverage Comparison Matrix</h2>
<table>
<thead>
<tr>
<th>Domain</th>
<th>CHMO</th>
<th>FBbi</th>
<th>OBI</th>
<th>ROR</th>
</tr>
</thead>
<tbody>
<tr>
<td>Analytical techniques</td>
<td>‚òÖ‚òÖ‚òÖ</td>
<td>‚òÖ</td>
<td>‚òÖ‚òÖ</td>
<td>-</td>
</tr>
<tr>
<td>Microscopy methods</td>
<td>‚òÖ</td>
<td>‚òÖ‚òÖ‚òÖ</td>
<td>‚òÖ‚òÖ</td>
<td>-</td>
</tr>
<tr>
<td>Sample preparation</td>
<td>‚òÖ‚òÖ</td>
<td>‚òÖ‚òÖ‚òÖ</td>
<td>‚òÖ</td>
<td>-</td>
</tr>
<tr>
<td>Instruments/devices</td>
<td>‚òÖ</td>
<td>‚òÖ</td>
<td>‚òÖ‚òÖ</td>
<td>-</td>
</tr>
<tr>
<td>Facilities</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>‚òÖ‚òÖ‚òÖ</td>
</tr>
<tr>
<td>Probes/dyes</td>
<td>-</td>
<td>‚òÖ‚òÖ‚òÖ</td>
<td>‚òÖ</td>
<td>-</td>
</tr>
<tr>
<td>Assay types</td>
<td>‚òÖ</td>
<td>‚òÖ</td>
<td>‚òÖ‚òÖ‚òÖ</td>
<td>-</td>
</tr>
<tr>
<td>Investigation lifecycle</td>
<td>-</td>
<td>-</td>
<td>‚òÖ‚òÖ‚òÖ</td>
<td>-</td>
</tr>
</tbody>
</table>
<p>‚òÖ‚òÖ‚òÖ = Excellent, ‚òÖ‚òÖ = Good, ‚òÖ = Limited, - = Not covered</p>
<hr />
<h2 id="strengths-by-ontology">Strengths by Ontology</h2>
<p><strong>CHMO</strong>:
- ‚úÖ Core structural biology techniques
- ‚úÖ Analytical chemistry methods
- ‚úÖ Simple, focused scope</p>
<p><strong>FBbi</strong>:
- ‚úÖ Multi-dimensional microscopy metadata
- ‚úÖ Comprehensive probe coverage
- ‚úÖ Sample preparation detail</p>
<p><strong>OBI</strong>:
- ‚úÖ Investigation context
- ‚úÖ Assay classification
- ‚úÖ Experimental design</p>
<p><strong>ROR</strong>:
- ‚úÖ Facility identification
- ‚úÖ Institutional metadata</p>
<hr />
<h2 id="gaps-limitations">Gaps &amp; Limitations</h2>
<p><strong>CHMO</strong>:
- ‚ùå Limited microscopy detail
- ‚ùå No facility information</p>
<p><strong>FBbi</strong>:
- ‚ö†Ô∏è Inactive (no updates since 2020)
- ‚ùå Missing cryo-EM, SAXS, crystallography
- ‚ùå No synchrotron terms</p>
<p><strong>OBI</strong>:
- ‚ùå Abstract device terms
- ‚ùå Limited technique detail
- ‚ö†Ô∏è Complexity overhead</p>
<p><strong>ROR</strong>:
- ‚ö†Ô∏è Organizations only, not techniques</p>
<hr />
<h2 id="current-usage-in-lambda-ber-schema">Current Usage in lambda-ber-schema</h2>
<p><strong>Implemented</strong>:
- ‚úÖ <strong>CHMO</strong> for TechniqueEnum (8 techniques)
- ‚úÖ <strong>CHMO</strong> for ExperimentalMethodEnum
- ‚úÖ <strong>ROR</strong> for FacilityEnum (12 facilities)
- ‚úÖ <strong>Wikidata</strong> for supplementary facility metadata</p>
<p><strong>Not yet implemented</strong>:
- ‚è∏Ô∏è FBbi for detailed microscopy workflows
- ‚è∏Ô∏è OBI for assay/protocol classification</p>
<hr />
<h2 id="recommendations">Recommendations</h2>
<p><strong>For structural biology techniques</strong>:
‚Üí <strong>CHMO</strong> (already implemented) ‚úÖ</p>
<p><strong>For facility identification</strong>:
‚Üí <strong>ROR</strong> (already implemented) ‚úÖ</p>
<p><strong>For detailed microscopy workflows</strong>:
‚Üí Consider <strong>FBbi</strong> if needed</p>
<p><strong>For investigation context</strong>:
‚Üí Consider <strong>OBI</strong> for assay classification</p>
<p><strong>For sample types</strong>:
‚Üí Consider <strong>NCBITaxon</strong>, <strong>CL</strong> (Cell Ontology)</p>
<hr />
<h2 id="future-enhancements">Future Enhancements</h2>
<p><strong>Potential additions</strong>:</p>
<ol>
<li><strong>FBbi mappings</strong> for microscopy-specific enums</li>
<li>GridTypeEnum ‚Üí FBbi grid types</li>
<li>
<p>VitrificationMethodEnum ‚Üí FBbi fixation methods</p>
</li>
<li>
<p><strong>OBI mappings</strong> for workflow/assay types</p>
</li>
<li>
<p>WorkflowTypeEnum ‚Üí OBI assay terms</p>
</li>
<li>
<p><strong>NCBITaxon</strong> for expression systems</p>
</li>
<li>
<p>ExpressionSystemEnum ‚Üí taxonomy IDs</p>
</li>
<li>
<p><strong>Additional ontologies</strong> for specialized domains</p>
</li>
</ol>
<hr />
<h2 id="integration-strategy">Integration Strategy</h2>
<p><strong>Multi-ontology approach</strong>:</p>
<pre><code class="language-yaml">cryo_em:
  description: &quot;Cryo-electron microscopy&quot;
  meaning: CHMO:0002413              # Primary: technique
  exact_mappings:
    - OBI:0003118                     # Assay context
    - FBbi:00000256                   # EM method (general)
  annotations:
    typical_facilities:
      - ROR:02jbv0t02                 # Berkeley Lab (ALS)
      - ROR:02ex6cf31                 # Brookhaven (NSLS-II)
</code></pre>
<p><strong>Benefits</strong>: Rich semantic connections across domains</p>
<hr />
<h2 id="best-practices">Best Practices</h2>
<ol>
<li><strong>Use <code>meaning:</code></strong> for primary ontology mapping</li>
<li><strong>Use <code>exact_mappings:</code></strong> for equivalent terms in other ontologies</li>
<li><strong>Use <code>broad_mappings:</code></strong> for hierarchical relationships</li>
<li><strong>Use <code>annotations:</code></strong> for domain-specific metadata</li>
<li><strong>Document</strong> ontology sources in comments</li>
<li><strong>Validate</strong> mappings using OAK or OLS</li>
<li><strong>Keep updated</strong> with ontology releases</li>
</ol>
<hr />
<h2 id="resources">Resources</h2>
<p><strong>Ontology browsers</strong>:
- OBO Foundry: http://obofoundry.org
- BioPortal: https://bioportal.bioontology.org
- OLS (EBI): https://www.ebi.ac.uk/ols4
- Ontobee: https://ontobee.org</p>
<p><strong>Tools</strong>:
- OAK (Ontology Access Kit): https://github.com/INCATools/ontology-access-kit
- ROBOT: http://robot.obolibrary.org</p>
<p><strong>Registries</strong>:
- ROR: https://ror.org</p>
<hr />
<h2 id="summary">Summary</h2>
<p><strong>Key takeaways</strong>:</p>
<ol>
<li><strong>CHMO</strong> provides excellent coverage for analytical techniques ‚úÖ</li>
<li><strong>FBbi</strong> offers rich microscopy detail but is inactive ‚ö†Ô∏è</li>
<li><strong>OBI</strong> gives investigation context and assay classification üî¨</li>
<li><strong>ROR</strong> enables persistent facility identification üè¢</li>
<li><strong>Multi-ontology approach</strong> maximizes semantic richness üåê</li>
</ol>
<p><strong>Current implementation</strong>: CHMO + ROR meets core needs</p>
<p><strong>Future</strong>: Consider FBbi/OBI for specialized workflows</p>
<hr />
<h2 id="questions">Questions?</h2>
<p><strong>Contact</strong>:
- lambda-ber-schema repository: https://github.com/lambda-ber/lambda-ber-schema
- Issues: https://github.com/lambda-ber/lambda-ber-schema/issues</p>
<p><strong>Generated</strong>: 2025-01-14
<strong>Schema version</strong>: Based on current main branch</p>







  
  






                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["content.tabs.link"], "search": "../../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
    
  </body>
</html>